Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/DeployToSqlServer.ps1

```diff
@@ -72,24 +72,29 @@ function GetScriptToRun
     $serverName = EscapeSpecialChars -str $serverName
     $databaseName = EscapeSpecialChars -str $databaseName
 
+    $securePasswordScript = "`$secureAdminPassword = `"$sqlPassword`" | ConvertTo-SecureString  -AsPlainText -Force"
```

Comment:
You don't need piping, you do this way :  ConvertTo-SecureString 'P@ssw0rd' -AsPlainText -Force